draft-cutler-httpbis-partitioned-cookies.md

```diff
@@ -85,8 +88,8 @@ This algorithm could be added after {{Section 5.2 ("Same-site" and "cross-site"
 
 {:quote}
 > 1. Let top-document be the active document in document's browsing context's top-level browsing context.
-> 2. Let "cookie-partition-key" be the site of the top-document when the user agent made the request.
-> 3. If the cookie is being read or written via a "non-HTTP" API, then cookie-partition-key is the site (as defined in {{HTML}}) of the top-document of the document associated with the non-HTTP API.
+> 2. Let "cookie-partition-key" be a tuple whose first element is the site of the top-document when the user agent made the request and the second element is a bit indicating if the active document has a cross-site ancestor.
```

Comment:
This is, admittedly, a bit of an informal definition of the ancestor-chain bit. Considering the target of this goal is to merge this into the cookie spec, RFC6265bis, perhaps we can reference this section and determine the boolean based on that result: https://datatracker.ietf.org/doc/html/draft-ietf-httpbis-rfc6265bis-13#section-5.2